src/containers/Login/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component, FormEvent } from 'react';
+import { FormGroup , ControlLabel, FormControl, Button, FormControlProps } from 'react-bootstrap';
+import './Login.css';
+
+import { Auth } from "aws-amplify";
+
+interface ILoginProps {
+  email: '',
+  password: ''
+}
+
+interface ILoginFormState {
```

Comment:
how are you planning to use the `Interface`? at this point of time `Type` would be enough right?